.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -26,35 +26,10 @@ set -e
 # directory and configuration directory.
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 CONF_DIR="${THIS_DIR}/../../../config"
-
-VAGRANT=0
-if [ -d "${THIS_DIR}/../.vagrant" ]; then
-    VAGRANT=1
-fi
-
-UTM=0
-if [ -d "${THIS_DIR}/../.utm" ]; then
-    UTM=1
-fi
-
-CI=0
-if [ -f "${THIS_DIR}/../.github_actions_ci_flag" ]; then
-    CI=1
-fi
-
 SUBMITTY_REPOSITORY=$(jq -r '.submitty_repository' "${CONF_DIR}/submitty.json")
-SUBMITTY_INSTALL_DIR=$(jq -r '.submitty_install_dir' "${CONF_DIR}/submitty.json")
-WORKER=$([[ $(jq -r '.worker' "${CONF_DIR}/submitty.json") == "true" ]] && echo 1 || echo 0)
 
-source "${THIS_DIR}/bin/versions.sh"
+source "${SUBMITTY_REPOSITORY}/.setup/install_submitty/get_globals.sh"
```

Comment:
why don't we do:

source "$( dirname "${BASH_SOURCE[0]}" )/get_globals.sh"

Like you have in line 4 of setup_directories.sh?

Reply:
`setup_directories.sh` and `INSTALL_SUBMITTY_HELPER.sh` are in different directories, so this would not find the file correctly as `BASH_SOURCE` differs. I'm using `SUBMITTY_REPOSITORY` here to get an absolute path instead of a relative one.